src/components/ListOptions/styles.module.scss

```diff
@@ -0,0 +1,10 @@
+@import '../../theme/variables.scss';
+
+ion-icon {
+  display: block;
+  font-size: 100%;
+  color: red;
+  margin-right: 10px;
```

Comment:
how about using `em` or `rem` so that margin adapts to view zoom or global font size?

Reply:
rem it is